Milvus.Client.Tests/CollectionTests.cs

```diff
@@ -287,8 +287,13 @@ public async Task Compact()
             });
 
         long compactionId = await collection.CompactAsync();
-        Assert.NotEqual(0, compactionId);
+        if ((await Client.GetVersionAsync()).StartsWith("v2.4.", StringComparison.Ordinal))
+        {
+            // Milvus 2.4 returns -1 here as the compaction ID
```

Comment:
@yhmo @weianweigan FYI, this is the only case where a test failed when transitioning from 2.3.10 to 2.4.0-rc.1 - not sure if this is intentional...

Reply:
Let me try to contact @yhmo .